src/test/java/org/openrewrite/java/migrate/apache/commons/lang/ApacheCommonsStringUtilsTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.migrate.apache.commons.lang;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+class ApacheCommonsStringUtilsTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.parser(JavaParser.fromJavaVersion().classpath("commons-lang3"));
+    }
+
+    @Test
+    void defaultString() {
+        rewriteRun(
+          spec -> spec.recipe(new DefaultStringRecipe()),
+          //language=java
+          java("""
+            import org.apache.commons.lang3.StringUtils;
+                          
+            class Foo {
+                String in = "foo";
+                String out = StringUtils.defaultString(in);
+            }
+            """, """
+            import java.util.Objects;
```

Comment:
This import isn't added presently; Instead the old import is retained. Any quick thoughts there @knutwannheden ?

Reply:
Possibly we only handle imports for the template and not for the LST. I need to take a look at the code. Can you tell from the generated class?

Reply:
I see import in the generated JavaTemplate before/after classes
```java
public class DefaultStringRecipe$1_after {
    public static JavaTemplate.Builder getTemplate(JavaVisitor<?> visitor) {
        return JavaTemplate
                .builder("Objects.toString(#{any(java.lang.String)})")
                .imports("java.util.Objects");
    }
}
```


I did not see imports added or removed in 
```java
return new JavaVisitor<ExecutionContext>() {
    final JavaTemplate before = JavaTemplate.compile(this, "before", (JavaTemplate.F1<?, ?>) (String s) -> defaultString(s)).build();
    final JavaTemplate after = JavaTemplate.compile(this, "after", (JavaTemplate.F1<?, ?>) (String s) -> Objects.toString(s)).build();

    @Override
    public J visitMethodInvocation(J.MethodInvocation elem, ExecutionContext ctx) {
        JavaTemplate.Matcher matcher;
        if ((matcher = before.matcher(getCursor())).find()) {
            return after.apply(getCursor(), elem.getCoordinates().replace(), matcher.parameter(0));
        }
        return super.visitMethodInvocation(elem, ctx);
    }

};
```


It's not pretty, but this gets the job done: https://github.com/openrewrite/rewrite-templating/pull/10

Reply:
I believe my plan was to use `ShortenFullyQualifiedTypeReferences` and another recipe to remove unused imports after applying the template. But I never got around to it.